Documentation/diagnostics/source_link.md

```diff
@@ -7,41 +7,98 @@ Most debugging is done against locally built source code on a developer's machin
 ## Source Link File Specification
 The `/sourcelink:<file>` compiler flag will embed a JSON configuration file in the PDB. This configuration file would be generated as part of the build process. The JSON configuration file contains a simple mapping of local file path to URL where the source file can be retrieved via http or https. A debugger would retrieve the original file path of the current location from the PDB, look this path up in the Source Link map, and use the resulting URL to download the source file.
 
-### MSBuild Example
-There is an open source tool available for generating Source Link configuration at build time. See https://github.com/ctaggart/SourceLink for the most up to date information.
+## MSBuild Example
+There is an open source tool available for generating Source Link configuration at build time. See https://github.com/ctaggart/SourceLink#quick-start for the most up to date information.
```

Comment:
After the first sentence, should we add "This supports GitHub and BitBucket out of the box"?